src/platform/forms-system/test/js/actions.unit.spec.js

```diff
@@ -312,6 +312,7 @@ describe('Schemaform actions:', () => {
         {
           fileTypes: ['jpg'],
           maxSize: 5,
+          maxPdfSize: 20,
```

Comment:
What are we testing by adding `maxPdfSize` to a `jpg` file?

Reply:
Making sure that the `maxPdfSize` doesn't influence a non-PDF file size limit